aleemer/Internify
# [IN-97] Make Delete call button in View page

#### User Story
[[IN-97] Make Delete call button in View page](https://chaosneutral.atlassian.net/browse/IN-97)

#### Changes made
- Added Delete button to the View page
- Added DELETE request call to the View page to delete a job posting
  - Delete to Job Posting model is followed by a callback to find and update the list of job postings from the User model
- Cleaned up some code

**NOTE**: I was working off of home page ticket when I was working on this task. You might see some code committed for the home page.

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
1. Go to the View page of a job posting
2. Delete a job posting
3. You should be redirected back to the profile page
